op-node/rollup/types.go

```diff
@@ -322,6 +322,42 @@ func (c *Config) IsInterop(timestamp uint64) bool {
 	return c.InteropTime != nil && timestamp >= *c.InteropTime
 }
 
+// ForkchoiceUpdatedVersion returns the EngineAPIMethod suitable for the chain hard fork version.
+func (c *Config) ForkchoiceUpdatedVersion(attr *eth.PayloadAttributes) eth.EngineAPIMethod {
```

Comment:
if `attr == nil`, we can return `eth.FCUV3`

Reply:
Applied: https://github.com/ethereum-optimism/optimism/pull/9253/commits/81841564d6dabcc728e64273ed318d3bf06003c7